tests/DbTestCase.php

```diff
@@ -455,4 +458,101 @@ protected function disableCapacity(
 
         return $definition;
     }
+
+    /**
+     * Helper method to help creating complex forms using the FormBuilder class.
+     *
+     * @param FormBuilder $builder RuleConfiguration
+     *
+     * @return Form Created form
+     */
+    protected function createForm(FormBuilder $builder): Form
```

Comment:
I guess form specific should not be part of main DbTest case, a FormTest Case should be added